group_vars/all/apt.yml

```diff
@@ -60,9 +60,10 @@ rabbitmq_erlang_repo: "{{ rpco_apt_repo }}"
 rabbitmq_gpg_keys: "{{ rpco_apt_gpg_keys }}"
 
 # ceph_client wiring
-ceph_apt_repos:
-  uca: "{{ rpco_apt_repo }}"
-ceph_pkg_source: "uca"
+# We use 'distro' to avoid any apt repo being setup by the ceph roles.
+# The repository needed (RPC-O apt artifacts repo) will already be on
+# the host.
+ceph_pkg_source: "distro"
```

Comment:
@mattt416 That's not something we can answer at this time. Once there is a storage product to test against, that question can be figured out and a solution put together. For now if we don't do this we end up with a rogue repo configured.

Note that from a product standpoint there is no support for Ceph for Newton+ until there is a ceph-based storage product developed, so this is a bit of a no-op which is only here to help the artifacts get configured right when executing tests.